### ISSUE TITLE
Error trying parse HTML

### ISSUE THREAD

**Issue by thiagogsr** [aknuds1/html-to-react#59]
Hi

When I tried parse it:

```html
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
<html>
  <body>
    <div style="text-align: center;"><img id="img_preview" src="https://....png" width="238" height="137"></div>
    <div style="text-align: left;">text</div>
  </body>
</html>
```

Into a `div`:

```jsx
import { Parser } from "html-to-react"

const X = ({ content }) = (
  <div>{new Parser().parse(content)}</div>
)
```

I received an error:

```
Uncaught DOMException: Failed to execute 'createElement' on 'Document': The tag name provided ('!doctype') is not a valid name.
```

Can you help me?

Thanks

**Comment by aknuds1** [comment]
react-to-html doesn't currently handle `doctype` directives, you can work around it by dropping the doctype directive at the beginning of your HTML.

**Comment by aknuds1** [comment]
I released a new version, 1.3.2, which ignores `doctype` directives in source HTML, can you please try this?
